src/app/modules/main/chat_section/module.nim

```diff
@@ -1406,7 +1404,7 @@ proc addOrUpdateChat(self: Module,
   if chat.id == activeChatId:
     self.updateActiveChatMembership()
 
-  if chatExists:
+  if self.doesCatOrChatExist(chat.id):
```

Comment:
Cat?

Reply:
Just the English version of "chat" :wink: